cloud-controller-manager/do/loadbalancers.go

```diff
@@ -439,60 +446,78 @@ func buildForwardingRules(service *v1.Service) ([]godo.ForwardingRule, error) {
 	certificateID := getCertificateID(service)
 	tlsPassThrough := getTLSPassThrough(service)
 
-	if (certificateID != "" || tlsPassThrough) && len(tlsPorts) == 0 {
+	if len(tlsPorts) == 0 && (protocol == protocolHTTPS || certificateID != "" || tlsPassThrough) {
```

Comment:
Why do we need the check for the HTTPS protocol? Why wouldn't it suffice to check for the other conditions as before?  🤔